cmdstanpy/cmdstan_args.py

```diff
@@ -277,6 +277,38 @@ def compose(self, idx: int, cmd: str) -> str:
         return cmd
 
 
+class GenerateQuantitiesArgs(object):
```

Comment:
this object takes a StanFit object as input and assembles the set of csvfiles used as `fitted_params` argument to the CmdStan `generate_quantities` method.
it runs the same number of chains as were run by the sample method.